pkgs/applications/version-management/git-and-tools/subgit/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, unzip, makeWrapper, jre }:
+
+stdenv.mkDerivation {
+  name = "subgit" + meta.version;
+
+  meta = {
+    description = "A tool for a smooth, stress-free SVN to Git migration";
+    longDescription = "Create writable Git mirror of a local or remote Subversion repository and use both Subversion and Git as long as you like. You may also do a fast one-time import from Subversion to Git.";
+    homepage = http://subgit.com;
+    license = stdenv.lib.licenses.unfree;
+    platforms = stdenv.lib.platforms.all;
+    version = "3.1.0";
```

Comment:
It's highly uncommon for packages to have a `meta.version` attribute. Why is that attribute required? Why don't you just add the version to `$name` directly, like almost all other packages do?